R/write_ragged_timeseries_dsg.R

```diff
@@ -10,7 +10,7 @@
 #'to names of observation columns in \code{all_data}. Not required. Columns without 
 #'supplied long name are given a long name of empty string.
 #'
-#'@param data_unit Optional list of observation units.  Names must correspond
+#'@param data_units Optional list of observation units.  Names must correspond
```

Comment:
It's a list in the function Luke wrote, vector in the function I wrote. Will open an issue to address.